pkg/console/operator/operator.go

```diff
@@ -143,6 +143,12 @@ func (c *consoleOperator) Key() (metav1.Object, error) {
 	return c.operatorConfigClient.Get(api.ConfigResourceName, metav1.GetOptions{})
 }
 
+type configSet struct {
```

Comment:
In addition, we can collect all the configs into a set & pass this, to also make it easier to pipe configs through.

Reply:
Note they are deliberately not embedded.